### ISSUE TITLE
In ordering screen: More information about ordering thresholds and tolerances

### ISSUE THREAD

**Issue by JuliusR** [foodcoops/foodsoft#260]
There could be more clarity concerning:
- `unit_quantity` (currently only in bottom infobar)
- amount of already filled `unit_quantities` (all groups together)
- included tolerance in the filled `unit_quantities` (all groups together)

**Comment by josefkreitmayer** [comment]
We are in a quite small foodcoop, and sometimes it just does not make sense to order, as there is not enough people ordering. Is there some possibility to see, how much value is already ordered with one supplier?

**Comment by carchrae** [comment]
i agree with @josefkreitmayer - you can see the amount already ordered (in the current order) by hovering the mouse and it is shown in the bottom, but this is a bit hidden.

i would also like to see the foodcoop-adam UI in the master version.  i may help port, or re-implement something.  there are some serious usability issues, especially on mobile devices.

**Comment by wvengen** [comment]
@josefkreitmayer that's a good point. I've opened an issue in https://github.com/foodcoops/foodsoft-shop/issues/7 regarding this. Ideas welcome. What do you say about the user story?

@carchrae Most work has already been done in [foodsoft-shop](https://github.com/foodcoops/foodsoft-shop) (together with an API for Foodsoft - PR #429), and its issues describe what's pending. Could really use some help here (I'm a bit stuck on optimistic updates), and I'm also curious to see what you'd think of the current UI (if you'd get it all running).

**Comment by carchrae** [comment]
@wvengen - cool. i will check it out.  i used spree commerce's api before, in a very similar architecture putting a lot more logic in the browser to make it a very fast/responsive UI, although i used angular not react.
